lib/stat_tracker.rb

```diff
@@ -121,5 +120,18 @@ def most_goals_scored(team_id)
   def least_goals_scored(team_id)
     GameTeam.least_goals_scored(team_id)
   end
+  
+  def favorite_opponent(team_id)
+    team = Team.all.find do |team|
+      team.team_id == GameTeam.favorite_opponent_id(team_id)
+    end
+    team.team_name
+  end
```

Comment:
it was pretty fun using a method from one class to get information from another class